pvlib/tracking.py

```diff
@@ -6,8 +6,10 @@
     PVSystem, Array, SingleAxisTrackerMount, _unwrap_single_value
 )
 from pvlib import irradiance, atmosphere
+from pvlib._deprecation import deprecated
 
 
+@deprecated('0.9.0', alternative='PVSystem with SingleAxisTrackingMount')
```

Comment:
hmm seeing the difference with the class name has me wondering... `SingleAxisTrackingMount` or `SingleAxisTrackerMount`?

Reply:
I surveyed n=2 pvlib users and both preferred `SingleAxisTrackerMount`.  I will make the change.

Reply:
Turns out it was already called `SingleAxisTrackerMount` and I just called it the wrong thing in the deprecation message 🤦‍♂️

Reply:
At least we got some data out of it!